OamLog.yaml

```diff
@@ -245,6 +265,7 @@ paths:
                         - subscriber-application
                         - subscriber-release-number
                         - subscriber-operation
+                        - subscriber-protocol
                         - subscriber-address
                         - subscriber-port
                       properties:
```

Comment:
line 275 :: PromptForBequeathingDataCausesRObeingRequestedToNotifyApprovalsOfNewApplicationsToNewRelease :: subscriber-application :: description :: Can we update the description to "Application Name of NewRelease" and it's corresponding path pointing to "http-client/application-name of NewRelease".
Because, if an application is having a new name other than http-server/application-name, the subscription needs to be made to new Release's application-name instead of current server application's name .

Reply:
Changed as proposed.

Reply:
Thanks for updating the path.

Also, the first line of description shall be changed to "Future application name"

Reply:
This comment is somehow missed to update. Kindly requesting for a check here.

Reply:
Changed description to _Application Name of NewRelease_

Reply:
Thanks for updating